### ISSUE TITLE
Add ability to check injection not only for GET parameters

### ISSUE THREAD

**Issue by TimurNurlygayanov** [s0md3v/XSStrike#145]
**Is your feature request related to a problem? Please describe.**
The current version can't detect xss injections if there is no get parameters in URL.
For example, I have an URL which works as a proxy and opens other web pages inside my page.
The URL of such page looks like: http://mysite.com/uuid1/uuid2 , where uuid1 and uuid2 is vulnerable, because if someone will add something like  http://mysite.com/http://attack.com{{}} - my app will proxy such request and it is also possible to write some data using this proxy.

**Describe the solution you'd like**
Need to try to apply XSS for all parts of URL after the domain name, like:  http://mysite.com/<XSS injection here>  - even we don't have GET parameters in this string.

**Comment by s0md3v** [comment]
Do you mean injecting payloads like this?
`http://example.com/search/<payload>`

**Comment by TimurNurlygayanov** [comment]
Yes, something like this (and without crawling - only for one page)

**Comment by s0md3v** [comment]
Yea, that's a case of URL rewriting and I was going to implement it but the main focus for now is to enhance the core capabilities.
You can expect an implementation soon.

**Comment by s0md3v** [comment]
That's DOM XSS
